spec/getinline/transformer_spec.rb

```diff
@@ -45,5 +45,27 @@
         end
       end
     end
+
+    context 'with special characters' do
+      let(:raw_with_special_characters) { File.read('spec/fixtures/raw/with-encoded-chars.erb') }
+      let(:text_with_special_characters) { File.read('spec/fixtures/text/with-encoded-chars.erb') }
+      let(:inlined_with_special_characters) { File.read('spec/fixtures/inlined/with-encoded-chars.erb') }
+
+      context 'with html tags' do
+        let(:transformer) { described_class.new(raw_with_special_characters) }
+        let(:transformed_html) { transformer.transform }
+        it 'encodes special characters to ASCII' do
+          expect(transformed_html).to include(inlined_with_special_characters)
```

Comment:
heeeey. When Jeremy originally wrote the specs, Nicholas suggested removing the `tranformed_html`let and calling it in the spec itself. So instead it'd be something like:
`context 'with html tags' do 
end`